app/Models/CreditStatus.php

```diff
@@ -0,0 +1,16 @@
+<?php
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+
+class CreditStatus extends Model
+{
```

Comment:
is it efficient to have a seperate `Credit Status` table? Why not just use the `credit status` field in the users table whereby the value will change depending on the user's credit status.

Reply:
My thinking was that this makes it cleaner. So that we have consistency in the data being entered. i.e CreditStatus::BAD, in stead of having it entered as a string. We could have one admin saying credit status `not good` and another saying `bad`. Does this make sense?

Reply:
Yes makes sense, it will be cleaner though I was thinking it might be unnecessary to add an extra relationship to a table which contains pre-defined data. This is because in the future if any change is made to the structure of the models the relationships might get affected which will mean further restructuring. @bmulobi what is your view, do we keep the `Credit Status` table?